sockeye/arguments.py

```diff
@@ -526,6 +526,14 @@ def add_model_parameters(params):
     model_params.add_argument('--rnn-context-gating', action="store_true",
                               help="Enables a context gate which adaptively weighs the RNN decoder input against the "
                                    "source context vector before each update of the decoder hidden state.")
+    # TODO: At the moment LHUC is RNN specific. We should support other models as well.
+    model_params.add_argument('--lhuc',
+                              nargs="+",
+                              default=[],
+                              choices=C.LHUC_CHOICES,
+                              metavar="COMPONENT",
+                              help="Use lhuc, include an amplitude parameter to hidden units. Valid values: %s"
```

Comment:
We could also list "Vilar 2018" here to point people to the paper.

Reply:
+1
[nit]. Period missing :)
I think this docstring could be a bit more detailed.

Reply:
Also, given that this only works with RNN, we should fail if user specifies non-RNN encoder/decoder but sets lhuc to those, to avoid silent mistakes.
A good place for this could be `check_arg_compatibility` in `train.py`.